samples-lib/src/main/java/com/esri/arcgismaps/sample/sampleslib/DownloaderActivity.kt

```diff
@@ -224,59 +231,56 @@ abstract class DownloaderActivity : AppCompatActivity() {
                 emit(LoadStatus.FailedToLoad(it))
                 return@flow
             }
+            // set up the file at the download path
+            val destinationFilePath = File(provisionLocation.path, portalItem.name).toString()
+            // set up the download URL
+            val downloadURL =
+                "${portalItem.portal.url}/sharing/rest/content/items/${portalItem.itemId}/data"
             // get the data of the PortalItem
-            val portalItemData = portalItem.fetchData()
```

Comment:
Did you create an issue for this? we need to investigate why the usage fails.

Reply:
The parent issue of this PR (`kotlin/issues/1995`) was created when we noticed `fetchData()` will download all the data into memory, so it was unsuitable for large file downloads. 

Should there be an individual issue to update the `fetchData()` design?